testsuite/rotational-diffusion-aniso.py

```diff
@@ -0,0 +1,326 @@
+from __future__ import print_function
+import unittest as ut
+import numpy as np
+from numpy.random import random, seed
+import espressomd
+import math
+import tests_common as tc
+
+@ut.skipIf(not espressomd.has_features(["ROTATION",
+                                        "PARTICLE_ANISOTROPY",
+                                        "ROTATIONAL_INERTIA",
+                                        "DIPOLES"]),
+           "Features not available, skipping test!")
+class RotDiffAniso(ut.TestCase):
+    longMessage = True
+    # Handle for espresso system
+    es = espressomd.System(box_l=[1.0,1.0,1.0])
```

Comment:
could you please use the standard naming convention in the testsuite and samples: ```system = espressomd.System(...)```